packages/babel-plugin-transform-runtime/test/fixtures/use-options/useESModules-cjs-auto/output.js

```diff
@@ -1,10 +1,10 @@
-var _classCallCheck = require("@babel/runtime/helpers/classCallCheck");
+var _classCallCheck = require("@babel/runtime/helpers/classCallCheck").default;
```

Comment:
similarly here but for other reasons - this PR removes `useESModules` so this fixture can be removed